extensions/wikia/WikiFactory/Dumps/maintenance/DumpsOnDemandCron.php

```diff
@@ -63,7 +63,10 @@ public function execute() {
 
         $oDB->update(
             'dumps',
-            array( 'dump_completed' => wfTimestampNow() ),
+            array(
+                'dump_completed' => wfTimestampNow(),
+                'dump_compression' => '7zip' // keep in sync with compression format in runBackups.php
```

Comment:
If this needs to be kept in sync why not introduce a constant in `DumpsOnDemand` class?